tasks/fix-cat3.yml

```diff
@@ -90,26 +90,76 @@
       - tmp
 
 - name: "LOW | RHEL-07-021600 | PATCH | The file integrity tool must be configured to verify Access Control Lists (ACLs)."
-  command: "true"
-  changed_when: no
+  block:
+    - name: "Replace CONTENT with CONTENT_EX"
+      replace:
+        path: /etc/aide.conf
+        regexp: '^(.*) (CONTENT)$'
+        replace: '\1 CONTENT_EX'
+      register: result
+      failed_when:
+        - result | failed
+        - result.rc != 257
+    - name: "Replace NORMAL with DATAONLY"
+      replace:
+        path: /etc/aide.conf
+        regexp: '^(.*) (NORMAL)$'
+        replace: '\1 DATAONLY'
+      register: result
+      failed_when:
```

Comment:
It might be worth it to have a `stat` task in this block and the same in the block in #88 to check if `/etc/aide.conf` exists and then only run these tasks if it exists. That would avoid the custom `failed_when` here.